app/services/to_cocina/work/types_mapper.rb

```diff
@@ -28,7 +28,7 @@ def call
       delegate :work_type, to: :work_form
 
       def work_subtypes
-        @work_subtypes ||= work_form.work_subtypes + [work_form.other_work_subtype].compact
+        @work_subtypes ||= work_form.work_subtypes + [work_form.other_work_subtype.presence].compact
```

Comment:
Not blocking, just a question: do you think this logic could live in the form itself or does it make more sense to stay here in the mapper?

Reply:
I could go either way, but slightly feel that blank is the correct value for the form (instead of nil).